python/test/unit/fem/test_assembler.py

```diff
@@ -357,34 +356,41 @@ def test_matrix_assembly_block(self, mode):
 
         def blocked():
             """Monolithic blocked"""
-            A = petsc_assemble_matrix_block(a_block, bcs=[bc])
+            A = petsc_assemble_matrix(a_block, bcs=[bc])
```

Comment:
Couldn't this now be simplified to `from dolfinx.fem.petsc import assemble_matrix? Is there a reason for the renaming?